server/views/partials/head.pug

```diff
@@ -10,4 +10,5 @@ head
   // Custom fonts
   link(href='https://fonts.googleapis.com/css?family=Lato:400,700', rel='stylesheet', type='text/css')
   // Custom CSS
-  link(rel='stylesheet', href='css/bootstrap-overrides.css')
\ No newline at end of file
+  link(rel='stylesheet', href='/css/bootstrap-overrides.css')
+  script(src="https://cdn.jsdelivr.net/npm/vue@2.5.17/dist/vue.js")
```

Comment:
Could we move this to a scripts partial and include it at the bottom of the body before we have any custom scripts? I wouldn't want any render blocking happening whilst this script loads and parses.